CHANGELOG.md

```diff
@@ -1197,7 +1197,7 @@ replace github.com/confio/ics23/go => github.com/cosmos/cosmos-sdk/ics23/go v0.8
 ### Bug Fixes
 
 * Implement dragonberry security patch.
-    * For applying the patch please refer to the [RELEASE NOTES](./RELEASE_NOTES.md)
+    * For applying the patch please refer to the [RELEASE NOTES](https://github.com/cosmos/cosmos-sdk/releases)
```

Comment:
i dont think this is the right link, can we lave a comment to ignore this link?

Reply:
Yep, should be https://github.com/cosmos/cosmos-sdk/releases/tag/v0.46.3